SubnetTree.cc

```diff
@@ -300,6 +300,51 @@ PyObject* SubnetTree::lookup(int family, inx_addr addr) const
     return data;
 }
 
+PyObject* SubnetTree::search_all(const char *cidr) const
```

Comment:
For completeness, should this support binary mode like `lookup()` does?

Reply:
@awelzel would we need a `size` param in that case, like with `lookup`, to determine the family?

Reply:
I believe so. Basically, when in binarymode=True mode, `search_all()` with a bytestring from `inet_pton()` should work, too:

```
In [2]: socket.inet_pton(socket.AF_INET, "192.168.0.1")
Out[2]: b'\xc0\xa8\x00\x01'
```

Reply:
@awelzel pushed a new commit to support binary mode. Let me know how that looks to you.